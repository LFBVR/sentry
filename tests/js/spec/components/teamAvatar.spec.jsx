import React from 'react';
import {shallow} from 'enzyme';

import TeamAvatar from 'app/components/teamAvatar';

describe('TeamAvatar', function() {
  describe('render()', function() {
    it('renders', function() {
      let team = TestStubs.Team();
      let wrapper = shallow(<TeamAvatar team={team} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
