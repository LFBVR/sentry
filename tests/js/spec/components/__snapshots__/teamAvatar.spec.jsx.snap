// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamAvatar render() renders 1`] = `
<span
  className="avatar"
>
  <Tooltip
    title="Team Name"
  >
    <LetterAvatar
      displayName="Team Name"
      identifier="team-slug"
    />
  </Tooltip>
</span>
`;
